### PR TITLE
Include a link to the application dashboard

### PR DESCRIPTION
**Pull request by mehalshah**
image

### CODE REVIEW

**Comment by mehalshah** [comment]
https://trello.com/c/DX7WAmRe/207-i-can-navigate-to-the-application-dashboard-from-my-homepage

**Comment by mehalshah** [comment]
Going to make this for workshop organizers when we remove the permission block

File: dashboard/app/views/home/_homepage.html.haml [hunk 1]
```diff
@@ -48,6 +48,11 @@
     %a{href:'/pd/workshop_dashboard'}
       %h3
         Manage PD Workshops
+    -# Update this to workshop organizer
```
Comment (aoby) [inline dashboard/app/views/home/_homepage.html.haml]: nit: prefix with `TODO:` and optionally (preferred) your name in parentheses so we know who's on the hook

**Comment by tanyaparker** [comment]
[I can navigate to the Application Dashboard from my homepage](https://trello.com/c/DX7WAmRe/207-i-can-navigate-to-the-application-dashboard-from-my-homepage)

File: dashboard/app/views/home/_homepage.html.haml (hunk 1 above)
Comment (tanyaparker) [reply dashboard/app/views/home/_homepage.html.haml]: And you mean `workshop_organizer` OR `workshop_admin` right?

Comment (mehalshah) [reply dashboard/app/views/home/_homepage.html.haml]: Yes to both

**Pull request merged**